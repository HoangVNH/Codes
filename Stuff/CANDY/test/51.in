10
76
XXVVXXVDVXVXVVDVDXXVVDVDXVXXXDVXXDVDDDVVDDVDVXDXXVDVVDDXXVDXDXXVXVDXXDXVVXDV
57
XDDDDDXXVVXDXDVVXXXDVDDVXXDXVVVXVXDVDVXXDVDVDVXVXVVDXDVVD
88
XVDXDXDXVVXXXDVDXVXXVDXDDVDVXDDVVXVDXVDDXXVVDDVXDXVDXXXDXDXDVVDDVXXVVXXVXXVVVXDXVXDDDVXV
70
VVVXXVVXDVVDVVXVXXXXVVDDXDXXXDVXDVXXVXDVXXXXDDVXXVVDDXXXXVDVDDDXDDDVVX
91
DVDDDXXVVVXVDXXVXXDDXXDVXXVDVVDDXDDXVVXVVVVXDDVVDXVVXVVXXXXXXDVXDDXXDXVDVVXVXXVDDVXDVVXDVVV
97
XVXDDXVVDXXXVXVDDXXXDDDVXDXVVXVVVXXDVDVXVVVVDVDXDVVVDXXVVVXDXVDXVXVXDXVXVVXVXVVDVVXXXVVDDVVDXVXDD
97
DVXDVXDDVDXXDDVVDXXVDVDDDVDDDXXVDDVVDDVDDVXXXDVVXDVVVDXDVXVXVDVXDDXDXVXVDXXDDDVVVDVXXDDXVXDVDDXXX
73
XVDDVXDDXXVDXXXVDXDDXXXVDVXVXXXDVXDDXVXVDXXDDXXVVDDXVVDXVDVXDXVXVVVDDXDVD
74
XXVVVXXVDXXXDVXXVDDDVXDXDXXVDXXXXDXXXVVDVXXXVXXDVXXDVVVXXXVXXXVXDVXXXVXVVD
97
XVDXXXDDVVVXVXXDDVVXDVDDDXDXXVXXXDXXVXXXVXVXVVXXDDVXDXVVXXDXVXDVDXVXDDDDXXDXXVDVVXVDXVXDVVVVDXVXX
