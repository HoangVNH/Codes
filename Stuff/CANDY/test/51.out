75
56
87
69
90
96
96
72
72
96
